tic-tac-toe/tic-tac-toe.rb

```diff
@@ -0,0 +1,90 @@
+class TicTacToe
+  def initialize
+    puts "Player 1, what's your name?"
```

Comment:
typically initialize should only be initializing values, this looks like the job of a i/o parser